src/components/controlItem/controlItem.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+  export let value: any
```

Comment:
And `number` is tenuous, as it's going to be dumped in the dom...

Reply:
In WC land, yes. In Svelte land, it can be anything. You're probably right, though, that string/number is more appropriate.

Reply:
I don't think it can be because we're storing it in the DOM, so if it's something more complex (like an array or object) it'll get converted to a (not necessarily useful) string, like `[Object]`